back/engines/commercial/admin_api/app/jobs/admin_api/bulk_delete_users_job.rb

```diff
@@ -0,0 +1,11 @@
+class AdminApi::BulkDeleteUsersJob < ApplicationJob
+  def run(emails)
+    emails.each do |email|
+      user = User.find_by!(email: email)
+      DeleteUserJob.perform_later(user)
+    rescue StandardError => e
+      Sentry.capture_exception(e)
```

Comment:
Thanks for `find_by_cimail`, a really useful hint.

But I'm still in doubt about notifying Sentry. If it was an online request, we could show a message to admin about not found users immediately. But with background jobs, if something goes wrong, the admin will never know it without Sentry. And a lot can go wrong: host is not correct, a special character in host/email, etc. And it may happen that all the errors will be just silently ignored.

So, for now, I would still use Sentry notifications. And if we get too many false alarms, I would remove it in the future. Please, let me know what you think about it.

Reply:
Fixed `find_by_cimail`. And I've also added some specs https://github.com/CitizenLabDotCo/citizenlab/pull/999/commits/153a987bc435e9c56b66009966b630d3194654a9